lib/user_agent_parser/parser.rb

```diff
@@ -35,9 +39,23 @@ def parse(user_agent)
       parse_ua(user_agent, os, device)
     end
 
+    def patterns_path
+      patterns_paths.first
+    end
+    deprecate :patterns_path, :patterns_paths, 2020, 12
```

Comment:
Perhaps, date can be changed a little :)

Reply:
Updated to 2022 :)